src/krishnaCodingTest_1/AgeDifference.java

```diff
@@ -0,0 +1,46 @@
+package krishnaCodingTest_1;
+
+import java.util.Scanner;
+
+/*WAP to provide the difference between a Younger and an elder member of the family. Age is provided in the form of an array.
+
+Sample Input: 7 12 56 62 25
+Sample Output: 55
+*/
+public class AgeDifference {
+	int younderElderAgeDiff(int[] age) {
+		int maxage = age[0];
+		for (int index = 1; index < age.length; index++) {
+			if (maxage < age[index]) {
+				maxage = age[index];
+			}
+		}
+		System.out.println("max age " + maxage);
+		int minage = age[0];
+		for (int index = 1; index < age.length; index++) {
```

Comment:
Different for loop not require, you can handle this in single loop with if conditions